content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -354,6 +360,23 @@ For example:
 
 The response would look the same as for non dry run request, but the values of some generated fields may differ.
 
+#### Dry Run Authorization
+
+Write authroization is required for the modifying verb for dry-run access.
+
+(`POST`, `PUT`, `PATCH`, and `DELETE`)
```

Comment:
I'm not sure listing the http methods here is useful

Reply:
Oops, I meant to add this after "the modifying verb." Fixed my intent

Reply:
Actually nevermind -- I removed this after the rewording.